packages/gatsby-design-tokens/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "gatsby-design-tokens",
+  "version": "1.0.0-alpha.1",
```

Comment:
@fk can I release this as 1.0.0 or do you want a pre-release first?

Reply:
I guess it doesn't _really_ matter, but I expect some changes/cleanup to be happening still to call this a true 1.0.0. Any worries about/drawbacks in consuming this as `1.0.0-alpha.1`?

Reply:
@fk not at all — this hasn’t been published whatsoever right? I can just release it as a pre-release!